app/actions/DaemonActions.js

```diff
@@ -579,6 +585,12 @@ export const getDcrdLogs = () => {
     });
 };
 
+export const getDcrdLastLineLogs = () => () => new Promise((resolve, reject) => {
```

Comment:
```suggestion
export const getDcrdLastLineLogs = () => () => new Promise((resolve, reject) => 
```

then closing `}` can go